go.mod

```diff
@@ -4,6 +4,8 @@ go 1.23.5
 
 replace github.com/redpanda-data/benthos/v4 => github.com/DIMO-Network/benthos/v4 v4.0.0-20250123192050-6af7fb655e87
 
+replace github.com/DIMO-Network/model-garage => github.com/DIMO-Network/model-garage v0.4.7-0.20250207154429-5f344b6dfce6
```

Comment:
if you are testing in the future you don't have to use replace just change the version in the required sections to v0.4.7-0.20250207154429-5f344b6dfce6